docs/sources/enterprise/datasource_permissions.md

```diff
@@ -33,6 +33,8 @@ When permissions are enabled for a data source in an organization, you restrict
 
 <div class="clearfix"></div>
 
+> **Caution:** Enabling permissions for the default data source makes users not listed in the permissions unable to invoke queries. Panels using default data source will return `Access denied to datasource` error for that users.
```

Comment:
Can you change the caution text to "data source"? It should be two words, not one.